src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big">
+      <label for="search1" class="search-Label search">
+        <input id="search1" type="text"
+        data-qa="keypress" placeholder="Try &quot;Los Angeles&quot;"
+        class="input big">
```

Comment:
```suggestion
        class="input input--big">
```
`--big` это модификатор